modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -100,8 +100,14 @@ def generate_pdf # rubocop:disable Metrics/MethodLength
 
         def synchronous
           auto_claim = shared_submit_methods
-          # This kicks off the first of three jobs required to fully establish the claim
-          process_claim(auto_claim, false) unless Flipper.enabled? :claims_load_testing
+
+          unless Flipper.enabled? :claims_load_testing
+            pdf_generation_service.generate(auto_claim&.id,
+                                            veteran_middle_initial)
+          end
+          if auto_claim.status != ClaimsApi::AutoEstablishedClaim::ERRORED
```

Comment:
eventually we will probably want to set an errored status in the pdf_generation (on failure) and just raise an exception to prevent this from even being necessary, but that can be cleanup when we add https://jira.devops.va.gov/browse/API-36598. I don't know the exact scenario that we would proceed here? 

seems like it would require `pdf_string.empty?` which we should probably rework with 36598 as well 🤷

Reply:
@FonzMP I added a link to this comment in API-36598